### PR TITLE
Reindex in admin side

### PR DESCRIPTION
**Pull request by priyankavasoya**
@harishmk1805 
please check this

### CODE REVIEW

File: app/code/Dss/Reindex/view/adminhtml/layout/indexer_indexer_list_grid.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                    </item>
+                    <item name="change_mode_reindex" xsi:type="array">
+                        <item name="label" xsi:type="string" translate="true">Reindex Data</item>
+                        <item name="url" xsi:type="string">reindexdata/indexer/massReindexData</item>
+                    </item>
+                </argument>
+            </arguments>
+        </referenceBlock>
+        <referenceContainer name="content">
+            <block class="Dss\Reindex\Block\Backend\Grid\Indexer" name="content.bsscontent"/>
```
Comment (harishmk1805) [inline app/code/Dss/Reindex/view/adminhtml/layout/indexer_indexer_list_grid.xml:41]: remove bss

File: app/code/Dss/Reindex/Block/Backend/Grid/Indexer.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            var isCoreModuleEnabled = '{$this->helper->isCoreModuleEnabled()}';
+            require(['jquery', 'domReady!'], function($) {
+                'use strict';
+                if (Boolean(isCoreModuleEnabled) !== true) {
+                    $('#gridIndexer_massaction-select option[value=\"change_mode_reindex\"]').remove();
+                }
+
+                $('.bss-reindex-info').closest('.message-success.success').addClass('bss-hidden');
+                $('.bss-reindex-show').click(function () {
+                    if ($('.bss-reindex-info').length > 0) {
```
Comment (harishmk1805) [inline app/code/Dss/Reindex/Block/Backend/Grid/Indexer.php:59]: remove bss

**Review by harishmk1805 — CHANGES_REQUESTED** [review]
remove "bss" and use dss

File: app/code/Dss/Reindex/view/adminhtml/layout/indexer_indexer_list_grid.xml (hunk 1 above)
Comment (priyankavasoya) [reply app/code/Dss/Reindex/view/adminhtml/layout/indexer_indexer_list_grid.xml]: @harishmk1805 
removed

File: app/code/Dss/Reindex/Block/Backend/Grid/Indexer.php (hunk 2 above)
Comment (priyankavasoya) [reply app/code/Dss/Reindex/Block/Backend/Grid/Indexer.php]: @harishmk1805 
removed

**Review by harishmk1805 — APPROVED** [review]

**Pull request merged**